src/client/app/utils/getGraphColor.ts

```diff
@@ -20,12 +20,12 @@ const colors = [
  * @param {String} label Graph label to hash
  * @returns {String} Hex color code
  */
-export default function getGraphColor(label) {
+export default function getGraphColor(label: string): string {
 	let hash = 0;
 	if (label.length !== 0) {
 		for (let i = 0; i < label.length; i++) {
-			hash = label.charCodeAt(i) + ((hash << 5) - hash); // eslint-disable-line no-bitwise
-			hash &= hash; // eslint-disable-line no-bitwise
+			hash = label.charCodeAt(i) + ((hash << 5) - hash); // tslint:disable-line no-bitwise
+			hash &= hash; // tslint:disable-line no-bitwise
```

Comment:
I think bitwise operations are sometimes appropriate. Perhaps we could disable this tslint rule?

Reply:
I think they're _very_ rarely appropriate, but I'm open to disabling the rule.

Reply:
They are so rarely appropriate that we definitely should not disable the rule.